### PR TITLE
Fix MRO publication

### PR DESCRIPTION
**Pull request by beckyjackson**
As requested by @rvita - the publication for MRO is incorrect, it should be [An ontology for major histocompatibility restriction](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4709943/).

### CODE REVIEW

**Comment by beckyjackson** [comment]
Can we allow `https://pubmed.ncbi.nlm.nih.gov/` for PMIDs? It's strange to me that this fails.

**Comment by cthoyt** [comment]
Technically, yes, but this would sort of defeat the point of standardizing these URLs. I think a better step forward would be to open this dictionary up to allow just putting in the pmid key itself, then downstram consumption could figure out how to format that into a URI.

File: ontology/mro.md [hunk 1]
```diff
@@ -26,8 +26,8 @@ usages:
       - url: https://www.iedb.org/assay/1357035
         description: "Epitope ID: 59611 based on reference 1003499"
     publications:
-      - id: https://www.ncbi.nlm.nih.gov/pubmed/30357391
-        title: "The Immune Epitope Database (IEDB): 2018 update"
+      - id: https://pubmed.ncbi.nlm.nih.gov/26759709
```
Comment (cthoyt) [inline ontology/mro.md:29]:
```suggestion
      - id: https://www.ncbi.nlm.nih.gov/pubmed/26759709
 ```

**Comment by beckyjackson** [comment]
+1 to putting the PMID in instead

**Review by matentzn — APPROVED** [review]

**Comment by matentzn** [comment]
@bpeters42 is listed as contact, given you are all part of the same time I assume this does not require a specific review by the contact. See also https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1848 for making this more transparent.

**Pull request merged**